packages/matchbox/src/components/Slider/styles.js

```diff
@@ -0,0 +1,126 @@
+import { tokens } from '@sparkpost/design-tokens';
+import { StyledSlider } from './Slider';
+
+export const slider = props => {
+  let cursor = 'pointer';
+
+  if (props.disabled) {
+    cursor = 'not-allowed';
+  }
+
+  return `
+    padding: ${tokens.spacing_600} 0;
```

Comment:
I think total height of the slider is supposed to 40px
![Screen Shot 2020-03-02 at 3 10 37 PM](https://user-images.githubusercontent.com/3903325/75713538-ffd22280-5c97-11ea-83a6-88dced90f7e1.png)